library/consumer-proguard-rules.pro

```diff
@@ -5,5 +5,5 @@
   @com.google.gson.annotations.SerializedName <fields>;
 }
 
--keepclassmembers class com.wultra.android.powerauth.networking.data.** { *; }
--keepclassmembers class com.wultra.android.powerauth.networking.error.** { *; }
\ No newline at end of file
+-keepclasseswithmembers, allowobfuscation class com.wultra.android.powerauth.networking.data.** { *; }
```

Comment:
So this should keep all the classes  in those packages (and subpackages). Only allowing obfuscation.
For Gson, it better to use combination of two rules:
```
# keeps fiels in the class
-keepclassmembers class somepackage.** { <fields>; }
# keeps the class that would R8 in full mode throw away otherwise
-keep, allowobfuscation class comepakcage.**
```